modules/virt-networking-glossary.adoc

```diff
@@ -19,7 +19,10 @@ Multus:: a "meta" CNI plug-in that allows multiple CNIs to exist so that a pod o
 Custom resource definition (CRD):: a link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Kubernetes]
 API resource that allows you to define custom resources, or an object defined by using the CRD API resource.
 
-Network attachment definition:: a CRD introduced by the Multus project that allows you to attach pods, virtual machines, and virtual machine instances to one or more networks.
+Network attachment definition (NAD):: a CRD introduced by the Multus project that allows you to attach pods, virtual machines, and virtual machine instances to one or more networks.
```

Comment:
Looks good to me! One minor nit: should those be 'Network Attachment Definition', 'Node Network Configuration Policy' (or without spaces?). `Container Network Interface` has the start of each word capitalised.